src/components/views/voip/VideoView.js

```diff
@@ -108,14 +111,16 @@ module.exports = React.createClass({
                  document.mozFullScreenElement ||
                  document.webkitFullscreenElement);
         const maxVideoHeight = fullscreenElement ? null : this.props.maxHeight;
-
+        const localVideoFeedClasses = classNames("mx_VideoView_localVideoFeed",
+            {"mx_VideoView_localVideoFeed_flipped": UserSettingsStore.getSyncedSetting}
```

Comment:
did you want to actually call this function?

Reply:
yes...